.github/workflows/build.yml

```diff
@@ -22,3 +27,4 @@ jobs:
         -Dpostgres.password=${{secrets.POSTGRES_PASSWORD}}
         -Dhome.application.admin.username=${{secrets.HOME_APPLICATION_ADMIN_USERNAME}}
         -Dhome.application.admin.password=${{secrets.HOME_APPLICATION_ADMIN_PASSWORD}}
+        -B org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
```

Comment:
use `sonar:sonar` instead of `org.sonarsource.scanner.maven:sonar-maven-plugin:sonar`

Also please move `-B` right after `mvn`. I suggest also add `-V -e` like a common practice.

And a bit order the definitions so we get:
```
mvn -V -B -e
      clean install sonar:sonar
# profiles
     -Papi-tests-infrastructure
# properties
      -Dpostgres.password=${{secrets.POSTGRES_PASSWORD}}
      -Dhome.application.admin.username=${{secrets.HOME_APPLICATION_ADMIN_USERNAME}}
      -Dhome.application.admin.password=${{secrets.HOME_APPLICATION_ADMIN_PASSWORD}}

Reply:
Done.